### ISSUE TITLE
DnD eating onMouseDown events breaks dgrid selection

### ISSUE THREAD

**Issue by dylans** [SitePen/dgrid#1401]
As reported by @jjrv in https://github.com/dojo/dojo/issues/278:

I have a dgrid like `OnDemandGrid.createSubclass([ Keyboard, ColumnResizer ]).createSubclass([ Tree, Editor, DnD ])`

With `DnD` enabled, clicking on a cell doesn't move the "keyboard focus" there. Meaning arrow keys jump back to the previous cell selected using the keyboard or by clicking on a form element.

I tracked the issue to this repo, `dnd/Selector.js` [here](https://github.com/dojo/dojo/blob/bd9156b2fd49233278a81818925e800863a64d38/dnd/Selector.js#L198-L199) and [here](https://github.com/dojo/dojo/blob/bd9156b2fd49233278a81818925e800863a64d38/dnd/Selector.js#L271-L272).

It seems the mousedown event doesn't reach some relevant handler in dgrid. The `stopPropagation` in this repo was apparently intended to stop text selection, but dgrid or its mixins already handle that.

I don't know of any nice way to handle this special case with dgrid without possibly breaking something else. I simply used `dojo/aspect` to redirect the event's `stopPropagation` and `preventDefault` methods to blank functions before reaching `onMouseDown` in `Selector.js`. That fixes the problem for my case.

For the record, here's the (TypeScript) code for patching the event:

```TypeScript
import * as Source from 'dojo/dnd/Source';

function nop() {}

aspect.before((Source as any).superclass, 'onMouseDown', (e: MouseEvent) => {
	e.stopPropagation = nop;
	e.preventDefault = nop;
	return([ e ]);
});
```

**Comment by msssk** [comment]
This should be fixed in #1445

**Comment by msssk** [comment]
Closing this since #1445 has ben merged.
